articles/ai-services/speech-service/how-to-pronunciation-assessment.md

```diff
@@ -621,7 +621,7 @@ You can get pronunciation assessment scores for:
 
 Pronunciation assessment can provide syllable-level assessment results. Grouping in syllables is more legible and aligned with speaking habits, as a word is typically pronounced syllable by syllable rather than phoneme by phoneme.
 
-Pronunciation assessment supports syllable groups in `en-US` with IPA and in both `en-US` and `en-GB` with SAPI.
+Pronunciation assessment supports syllable groups only in `en-US` with IPA and in both `en-US` and `en-GB` with SAPI.
```

Comment:
The original is just poorly worded because of the order of reference.

I would have worded it as "Pronunciation assessment supports syllable groups using the IPA phonetic alphabet in en-US, and SAPI alphabet in en-US and en-GB."

Though even that is a bit lacking, as what about the non-english languages and locales, do they also support only IPA for some locals within a language, and both IPA and SAPI in others?  I think it would be better to create a table of all languages and locals, and then list what alphbet is supported for each locale...

Reply:
Totally agree. I would be happier if we could have a table here that looks something like:

| Locale | IPA | SAPI|
|---|---|---|
|en-US| x | x |
|en-GB| | x |
| .... | | |

Do you know would could produce that? I got added to into an email thread by Marko. I will try asking there, but if you have a good contact for this, I am more than happy to follow up on it.

Reply:
Just to close the information loop: I was able to get the information from the right team so I will adjust the PR accordingly